assignments/Assignment5.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script src="Assignment5.js"></script>
+</head>
+<body>
+    <input id="centerWidth" type="text" placeholder="centerWidth"></input><br>
+    <button onclick="diamondPatternOnClickHandler()">Get diamond pattern</button><br>
+    <input id="res" type="text"></input>
```

Comment:
This should be a `<textarea...>` instead:

https://www.w3schools.com/tags/tag_textarea.asp